src/game/layers.cpp

```diff
@@ -54,7 +56,22 @@ void CLayers::InitGameLayer()
 						m_pGameGroup->m_ClipH = 0;
 					}
 
-					return; // there can only be one game layer and game group
+					// return; // there can only be one game layer and game group
+				}
+				else if(pTilemap->m_Flags > TILESLAYERFLAG_GAME)  // this is some other
+				{
+					char name_buf[12];
+					IntsToStr(pTilemap->m_aName, sizeof(pTilemap->m_aName)/sizeof(int), name_buf);
+					if(str_comp_nocase(name_buf, "Material") == 0)  // save files shouldn't be localized
+					{
+						if(!m_pMaterialLayer)
+							m_pMaterialLayer = pTilemap;
+					}
+					else
+					{
+						// Here we could find other layers without changing any mapitems
+						m_apCustomLayers.add(pTilemap);
```

Comment:
@AssassinTee pls check on deletion, that you are removing this layer from that array, too!